presto-main/src/test/java/com/facebook/presto/block/TestInterleavedBlock.java

```diff
@@ -134,15 +134,16 @@ private void assertValues(Slice[] expectedValues)
         Type type = TYPES.get(position % TYPES.size());
         assertInterleavedPosition(ImmutableList.of(type), block.getSingleValueBlock(position), 0, expectedValue);
 
-        assertInterleavedPosition(ImmutableList.of(type), block.getRegion(position, 1), 0, expectedValue);
-        assertInterleavedPosition(TYPES, block.getRegion(0, position + 1), position, expectedValue);
-        assertInterleavedPosition(ImmutableList.of(type), block.getRegion(position, block.getPositionCount() - position), 0, expectedValue);
+        int positionFloored = position / COLUMN_COUNT * COLUMN_COUNT;
```

Comment:
`alignedPosition`, or `entryStartPosition`, maybe?

Also, either add clarifying parenthesis or rewrite it as `position - position % COLUMN_COUNT`